.github/workflows/main_workflow.yml

```diff
@@ -93,8 +93,8 @@ jobs:
         pip install --user --upgrade setuptools
 
         # install Protobuf compiler
-        wget https://github.com/protocolbuffers/protobuf/releases/download/v3.19.4/protoc-3.19.4-linux-x86_64.zip
-        unzip protoc-3.19.4-linux-x86_64.zip -d protoc
```

Comment:
Have you checked that this is the only reference to 3.14.4 in this file? I had lots of them

Reply:
actually there a few more places required, thanks